Sources/BraintreeVenmo/BTVenmoAppSwitchReturnURL.swift

```diff
@@ -68,6 +68,8 @@ struct BTVenmoAppSwitchReturnURL {
     /// - Parameter url: an app switch return URL
     /// - Returns: `true` if the url represents a Venmo Touch app switch return
     static func isValid(url: URL) -> Bool {
-        url.host == "x-callback-url" && url.path.hasPrefix("/vzero/auth/venmo/")
+        url.scheme?.caseInsensitiveCompare(BTAppContextSwitcher.sharedInstance.returnURLScheme) == .orderedSame
```

Comment:
This is called from `BTVenmoClient.handleReturnURL` and enforces URL schemes for returning to the app

Reply:
It doesn't have to but the ticket listed "Consider updating VenmoClient’s handleReturnURL methods to enforce URL scheme & disallow https / universal link" 😄

Reply:
I think I overthought that when I originally wrote it, I don't think this logic has to change!

Reply:
No worries, reverted here: https://github.com/braintree/braintree_ios/pull/1218/commits/a6aabdd86d68304e041a69481c000f1e7a9973e9